core/server/common/src/main/java/alluxio/master/journal/raft/transport/CopycatGrpcClient.java

```diff
@@ -78,8 +78,12 @@ public CopycatGrpcClient(AlluxioConfiguration conf, UserState userState,
     final CompletableFuture<Connection> buildFuture = CompletableFuture.supplyAsync(() -> {
       try {
         // Create a new gRPC channel for requested connection.
+        String hostName = address.socketAddress().isUnresolved() ? address.host()
+            : address.socketAddress().getAddress().getHostName(); // use unresolved hostname
```

Comment:
I guess we'll never get a resolved address here. 
Even if we could, why do you need a seperate handling?

Reply:
you're right, this is stale now, i'll remove this